src/components/search.vue

```diff
@@ -111,6 +111,10 @@ export default {
           this.reason = 'Les coordonnées du point sont introuvables'
         }
       }
+      for (let res of result) {
+        res.lng = Math.round(res.lng * 1000000) / 1000000
```

Comment:
https://gis.stackexchange.com/questions/8650/measuring-accuracy-of-latitude-and-longitude

C’est bon ! C’est assez de décimales :)